app/models/application_choice.rb

```diff
@@ -146,6 +146,10 @@ def study_mode_full?
     course_option.no_vacancies?
   end
 
+  def course_not_open_for_applications?
+    course.applications_open_from > Time.zone.today
+  end
+
```

Comment:
my preference would be to remove this method, having a method in `Course` instead:
```
def open_for_applications?
  applications_open_from <= Time.zone.today
end
```
and then replace the usage below (line 168) with `!course.open_for_applications?`

Reply:
🧹